acquisitions/src/test/java/org/folio/GobiApiTest.java

```diff
@@ -18,19 +16,9 @@ public GobiApiTest() {
         super(new TestIntegrationService(new TestModuleConfiguration(TEST_BASE_PATH)));
     }
 
-    @BeforeAll
-    public void gobiApiTestBeforeAll() {
-    runFeature("classpath:thunderjet/mod-gobi/gobi-junit.feature");
-    }
-
-    @AfterAll
-    public void financeApiTestAfterAll() {
-    runFeature("classpath:common/destroy-data.feature");
-}
-    // Feature(s) list :
```

Comment:
gobi is individual test compoentents but actually it is using consortia style test, so removed unnecessary steps